scripts/import/clinvar_post_process.pl

```diff
@@ -126,7 +126,7 @@ sub run_variation_checks {
         #the expectation is that any new rsID that has to be inserted is a new one (not in dbSNP import)
         #given that approx. 650mil variation is already known, the new rsID should have a bigger number than that
         my $number = $var{name}  =~ s/rs//r;
-        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if $number < 650000000;
+        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if $number < 650000000 if $DEBUG == 1;
```

Comment:
```suggestion
        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if($number < 650000000 and  $DEBUG == 1);
```

Reply:
```suggestion
        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if($number < 650000000 &&  $DEBUG == 1);
```
if we have a preference for && over and

Reply:
Thanks for pointing that out. This is mistake.